example/stable-diffusion/txt2img-ft/runtime.yaml

```diff
@@ -0,0 +1,11 @@
+api_version: 1.1
+dependencies:
+  - pip:
```

Comment:
remove `starwhale>=0.6.0`? 
and in the req.txt, `starwhale[online-serve] >= 0.6.0`

Reply:
done